routes/contentCreatorRoutes.js

```diff
@@ -1,31 +1,30 @@
-const router = require("express").Router();
-
+const router = require('express').Router();
+const errorCodes = require('../helpers/errorCodes');
 const express = require('express');
 const app = express();
 app.use(express.json());
-app.use(express.urlencoded({extended: true}));
+app.use(express.urlencoded({ extended: true }));
 
-// Models
-/*
 const {
-  ContentCreatorApplication,
+  ContentCreator,
 } = require("../models/ContentCreatorApplication");
 
-router.delete("/profile/delete/:id", async (req, res) => {
-    try {
-      // Get the authenticated creator's ID from req.creator.id
-      const { id } = req.params;
-  
-      // Use Mongoose to find and delete the user by ID
-      console.log("Deleting creator with ID:", id)
-      await UserModel.findByIdAndDelete(id);
-  
-      // Send a success response
-      res.status(200).json({ message: "Content creator deleted successfully" });
-    } catch (error) {
-      // Handle any errors and send an error response
-      console.error("Error deleting content creator:", error);
-      res.status(500).json({ error: "An error occurred while deleting the content creator" });
+router.delete("/profiles/:id", async (req, res) => {
```

Comment:
Do we need /profiles?
I think it may not be necessary.

So we can just use /api/creators/id format
Can we fix this? that is the only "flaw" i can find 😄 👍

Reply:
fixed